Sadot/User/UserHomePage.cs

```diff
@@ -15,11 +15,45 @@
 
 namespace Sadot
 {
+    enum ETableIDs
+    {
```

Comment:
This enum is unnecessary (even though we talked about enums), because it is classic for indeces

Reply:
What about the cases where the table IDs are not in ascending order?

Reply:
Removed